README.md

```diff
@@ -388,5 +388,6 @@ As the front end ecosystem grows, we are actively exploring, experimenting and e
 
 ### Footnotes
 
-[^1]: This can be solved via [webpack code splitting](https://webpack.js.org/guides/code-splitting/).
-[^2]: [Universal JS](https://medium.com/@mjackson/universal-javascript-4761051b7ae9) to the rescue!
+<a href="#ft-1" name="ft-1">\*</a> This can be solved via [webpack code splitting](https://webpack.js.org/guides/code-splitting/).
```

Comment:
I believe the markdown anchor syntax would work too. For reference, this is how it's supposed to look like: http://engineering.grab.com/grabs-front-end-study-guide